### PR TITLE
More robust checkboxes

### PR DESCRIPTION
**Pull request by jasonfellows**
Background-clear doesn't work as well as we'd like out in the wilds of HCM.

* Making background-color more specific and explicit
* Making tooltips look like tooltips without React

### CODE REVIEW

**Comment by loudhearted** [comment]
:+1:

File: src/scss/components/_tooltip.scss [hunk 1]
```diff
@@ -1,4 +1,7 @@
 .tooltip {
+  background-color: $blue-95;
+  border-radius: 6px;
+  padding: $space-2;
```
Comment (HipsterBrown) [inline src/scss/components/_tooltip.scss]: Why do the utility classes not work?

Comment (jasonfellows) [inline src/scss/components/_tooltip.scss]: This lib needs to support the use case where an app needs the styles, but not the React components. We need to decouple appearance from the components.

Comment (HipsterBrown) [inline src/scss/components/_tooltip.scss]: Ah ok, that makes sense. I don't like repeating styles but definitely a good reason to do so here. :+1:

**Comment by HipsterBrown** [comment]
:+1:

**Pull request merged**